packages/amplify-graphql-auth-transformer/src/__tests__/accesscontrol.test.ts

```diff
@@ -1,7 +1,7 @@
 import { AccessControlMatrix } from '../accesscontrol';
 import { MODEL_OPERATIONS } from '../utils';
 
-test('test access control on object and field', () => {
+test('access control on object and field', () => {
   /*
   given the following schema
   type Student
```

Comment:
can you please also update the schema here with new permissions?

Reply:
not sure I follow? the schema that has `read` permission should still work as before without customers to need to make any changes

Reply:
Yeah but in this test, the schema here reflects one constructed in the test and in the test we replace `read` with `get/list etc`. Want to keep the documented schema in parity with code changes

Reply:
updated